avocado/utils/partition.py

```diff
@@ -297,3 +297,22 @@ def unmount(self, force=True):
                 else:
                     raise PartitionError(self, "Unable to unmount gracefully",
                                          details)
+
+
+def run_dd(opf, count, ipf='/dev/zero', block_size='1M', args=''):
+    """
+    Run dd command for the specified arguments
+
+    :param opf: path to output file
+    :type opf: str
+    :param count: Number of loops i.e count value
+    :type count: int
+    :param ipf: path to input file
+    :type ipf: str
+    :param block_size: block size suffixed with memory unit
+    :type block_size: str
+    :param args: Other arguments to be passed to dd command
+    :type args: str
+    """
+    process.run('dd if=%s of=%s bs=%s count=%d %s'
+                % (ipf, opf, block_size, count, args))
```

Comment:
> So, if required we can provide the default value for all args as you mentioned, for ex by keeping bs value as 'size of page in system' .

It's up to you. The reason to have such function would be additional value when compared to plain `proces.run('dd...')`. If there's no additional value, there's no reason for this function to exist.

> And , ipf was used instead of 'if' keyword.

Yep, I figured that out, but I would not like to have to. I think every single test writer will think the same.